src/index.html

```diff
@@ -1,13 +1,59 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img
+        class="product-card__img"
+        src="images/imac.jpeg"
+        alt="iMac"
+      >
+      <div class="product-card__name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="product-card__code">
+        Код товара: 195434
+      </div>
+      <div class="product-card__rating">
+        <div class="product-card__stars">
+          <div class="product-card__star"></div>
+          <div class="product-card__star"></div>
+          <div class="product-card__star"></div>
+          <div class="product-card__star"></div>
+          <div class="product-card__star"></div>
+        </div>
```

Comment:
Stars block should be independent i guess, also star(stars block children) - bem element of the stars block

Reply:
Thank you! Fixed stars block.